### DISCUSSION TITLE
ForgeFed support

### DISCUSSION THREAD

**Discussion by RGBCube** [community/community#65178] [Apps, API and Webhooks]
Will GitHub support the ActivityPub extension, [ForgeFed](https://forgefed.org)?

Gitea and GitLab are working on their implementation, and it would be great if all major git platforms supported the protocol enhancing the federation.

**Comment by Mubelotix** [comment]
Well if they don't they die. Given the userbase github has, the fediverse will take over quickly

**Reply by RGBCube** [reply]
I don't think most people will move off of GitHub because it's just familiar and the free offerings are actually much better than other forgejo/gitea/sourcehut instances

**Reply by Mubelotix** [reply]
Nothing is more familiar and free than your home server. Devs will move. At least the best ones

**Reply by RGBCube** [reply]
Not everyone loves self-hosting services. I doubt even 0.5% of the platform would move off.
